docker/.env

```
@@ -17,15 +17,23 @@
 # - `oceanbase` (https://github.com/oceanbase/oceanbase)
 # - `opensearch` (https://github.com/opensearch-project/OpenSearch)
 # - `seekdb` (https://github.com/oceanbase/seekdb)
+# - `gaussdb` GaussDB Centralized / Distributed DocEngine
 DOC_ENGINE=${DOC_ENGINE:-elasticsearch}
 
+# The business metadata database type used by DB_TYPE.
+# Available options: mysql (default), postgres, gaussdb, oceanbase.
+DB_TYPE=${DB_TYPE:-mysql}
```

dotenv-linter (4.0.0)

[warning] [SubstitutionKey] The DB_TYPE key is not assigned properly

(SubstitutionKey)